data/json/items/tool_armor.json

```diff
@@ -2526,5 +2526,65 @@
     "turns_per_charge": 3,
     "use_action": { "type": "transform", "msg": "You turn the blanket's heating elements off.", "target": "electric_blanket" },
     "magazine_well": 2
+  },
+  {
+    "id": "foodperson_mask",
+    "type": "TOOL_ARMOR",
+    "name": "Foodperson mask",
+    "description": "Foodperson, the mascot your stomach deserves!",
+    "weight": 5000,
```

Comment:
5 kilograms mask?

Reply:
I went a bit overboard there. I just changed it to 1.5kg, it's a pretty big thing with electronic in for the lights and the sounds.